public/js/medid/util.js

```diff
@@ -0,0 +1,16 @@
+define(function() {
+  var util = {};
+
+  util.formatDate = function () {
```

Comment:
good that you separated this function, it was duplicated in card.js/document.js.